src/plugin/updatenotifier.h

```diff
@@ -58,12 +58,12 @@ class UpdateNotifier
     ~UpdateNotifier() override;
 
     void notify(MImUpdateEvent *event);
-    void notifyOverride(const Logic::KeyOverrides &overriden_keys,
+    void notifyOverride(const Logic::KeyOverrides &overridden_keys,
                         bool update = false);
 
     Q_SIGNAL void cursorPositionChanged(int cursor_position,
                                         const QString &surrounding_text);
-    Q_SIGNAL void keysOverriden(const Logic::KeyOverrides &overriden_keys,
+    Q_SIGNAL void keysOverriden(const Logic::KeyOverrides &overridden_keys,
```

Comment:
Shouldn't you also fix the signal name, and the connections where it's used, rather than just variable names? :)

Reply:
I'll update the patch in Debian's upload, as well.